app/models/conversation.rb

```diff
@@ -1,2 +1,3 @@
 class Conversation < ApplicationRecord
+  has_many :messages
```

Comment:
should also have =>
 has_many :user_conversations
 has_many :messages, through: :user_conversations

Reply:
done